### PR TITLE
YLP-1088 Rework UX/UI of the footer

### PR DESCRIPTION
**Pull request by TimCanu**
- Implemented new responsive design
- Language select in the footer is only shown when the user is not logged in
- Responsive design is triggered based on mui breakpoints

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ It is based on Tidepool Blip 1.27.
 
 ## Unreleased
 ### Added
+- YLP-1088 Rework UX/UI of the YLP footer
```
Comment (yanngit) [inline CHANGELOG.md:7]: is part of improvements ?

**Review by yanngit — APPROVED** [review]
Nice responsive handling, good job with css hell I'm just validating it works not that the css could be better (I can't do that I'm not a robot ... or css guru)

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Mobile view: yourloops by diabeloop could be placed better?
Portrait
image
Paysage:
image

File: packages/yourloops/lib/diabeloop-url.tsx [hunk 2]
```diff
@@ -82,8 +82,13 @@ class DiabeloopUrl {
   }
 
   getIntendedUseUrL(currentLangue: string): string {
-    this.intendedUseUrL = `${this.rootPathName}intended-use.${currentLangue}.pdf`;
-    return this.intendedUseUrL;
+    this.cookiesPolicyUrL = `${this.rootPathName}intended-use.${currentLangue}.pdf`;
```
Comment (yvon-dblg) [inline packages/yourloops/lib/diabeloop-url.tsx:85]: Something is strange here

**Review by yvon-dblg — CHANGES_REQUESTED** [review]

**Pull request merged**